src/auth/Config.h

```diff
@@ -20,13 +20,14 @@ namespace Auth
 
 class Config
 {
+    // no support for copying of any kind
     explicit Config(const Config &) = delete;
     explicit Config(const Config *) = delete;
```

Comment:
We don't care about type conversion constructors etc. The only reason there is any of this is because the devs implementing schemeAccess insisted on using a raw-pointer for the member instead of a proper CbcPointer, AND could not be bothered implementing the copy constructors that choice requires.

Reply:
Sorry, I do not see a relationship between my change request and your response. In fact, the only way I can interpret "we don't care" is "yes, I will delete these unnecessary lines", but the overall response does not sound like an agreement with my change request, so that interpretation is probably wrong.

Reply:
Sorry, to clarify the only constructors we care about are the explicit copy/move constructors. The move constructor prevents implicit copy being defined, but does not prevent copy being defined in scheme-specific child classes. Only delete does both.

Reply:
> The move constructor ... does not prevent copy being defined in scheme-specific child classes.

True. Nothing can prevent that AFAIK.

> Only delete does

I am not sure what you mean exactly, but the lines this change request wants to delete do not "prevent copy being defined in child classes"! The first one has no effect that I know about, and the second one just changes a very reasonable/clear "no matching function" error to a "use of deleted function" error.

Here is a simple test to prove that child classes can still define copy constructors even if their parent class contains the lines this change request asks to delete: https://godbolt.org/z/f1q8T6dWv

Reply:
Done in 8d7b645